### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by SOUx4cx4fx56x69**
You want windows? -> We may help to you.
This is now nano-technology of reptiloid.

### CODE REVIEW

**Comment by SOUx4cx4fx56x69** [comment]
If you are the love jumped from windows it that you need.

**Comment by SOUx4cx4fx56x69** [comment]
I everyday just jump from my windows, but i live there 1 level...

**Comment by SOUx4cx4fx56x69** [comment]
It does not metter, that shit can help to windows users compile that without cygwins

**Comment by SOUx4cx4fx56x69** [comment]
<R4SAS> vanity.hpp:35:8: error: 'cpu_set_t' does not name a type
<R4SAS>  static cpu_set_t * cpu;
<R4SAS> че за хуйня
"че за хуйня">>What the fuck?
He don't understand because in the windows don't exist cpu_set_t :(
But now that have to work

**Comment by SOUx4cx4fx56x69** [comment]
Pleaze, anyone check this.

**Comment by SOUx4cx4fx56x69** [comment]
Amen.

File: vanity.hpp [hunk 1]
```diff
@@ -10,19 +10,31 @@
...
 std::cout << "CPU_ALLOC error" << std::endl;\
 return 1;\
 }\
 }
-
-
-
+#else
+#warning don't testing
+#define INIT_CPUS(count_cpu){\
```
Comment (majestrate) [inline vanity.hpp]: why not make this a function?

Comment (SOUx4cx4fx56x69) [inline vanity.hpp]: @majestrate Religion doesn't allow me

**Pull request merged**